Readme.md

```diff
@@ -1,29 +1,27 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128592237/22.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E622)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# XAF - How to prevent a Detail View from being shown for a List View record
+# XAF - Prevent Showing a Detail View of a List View Object
 
-## Scenario
+This example demonstrates how to prevent a user from invoking a Detail View of an object from a List View.
 
-For certain data forms, a developer needs to limit end-users to editing only through the List View, i.e. without invoking a separate Detail View. This is usually done by activating the inline editing and `MasterDetailMode` = `ListViewAndDetailView` features as described in the following article: [List View Edit Modes](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes).
+To allow users to edit objects directly in a List View, enable the [in-place editing](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#in-place-editing) and [split layout](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#split-layout-the-masterdetailmode-property) functionality.
 
 ## Implementation Details
 
-To accomplish this task, access the [ListViewProcessCurrentObjectController class](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController) using the [Frame.GetController<ControllerType>() method](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Frame.GetController--1) and disable the ListViewProcessCurrentObjectController.ProcessCurrentObjectAction.
+Use the [Frame.GetController<ControllerType>](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Frame.GetController--1) method to access the [ListViewProcessCurrentObjectController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController) class and disable the [ListViewProcessCurrentObjectController.ProcessCurrentObjectAction](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController.ProcessCurrentObjectAction) property.
```

Comment:
This sentence has a potential technical clarity issue -- you say that one should use the `Frame.GetController` method to access the mentioned class and disable some property...

But thing is, that you are most likely accessing an instance of the mentioned class rather than the class itself. The latter would have been correct if you rather changed a class property value, when a class is not a template for creating objects from it but sort of a namespace. From what I see here, this is not the case.

However, I could have been wrong because I don't know inner workings of your code in this scenario. Please clarify. Technical correctness and clarity are extremely important. Thanks in advance.

Also, I urge you to favor more accurate verbs over more generic verbs. In this particular scenario, would like to see "call" or "invoke" instead of "use" when it's related to a method.

Reply:
This is indeed the case of accessing an instance of the class, Dmitry, you are correct, we should use more precise wording here.

```suggestion
Call the [Frame.GetController<ControllerType>](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Frame.GetController--1) method to access the [ListViewProcessCurrentObjectController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController) class and disable the [ListViewProcessCurrentObjectController.ProcessCurrentObjectAction](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController.ProcessCurrentObjectAction) property.
```

Reply:
Yes, this change works, thank you. As a possible alternative, you could use the following pattern:

> to access a ClassName object.